snuba/web/db_query.py

```diff
@@ -189,8 +212,12 @@ def execute_query_with_deduplication(
 
 
 def raw_query(
-    request: Request,
-    query: ClickhouseQuery,
+    clickhouse_query: Query,
+    request_settings: RequestSettings,
+    # TODO: Passing the SqlQuery (which basically wraps the query and formats it)
+    # will be needed as long as the legacy query representation will be around.
+    # See DictSqlQuery.
+    formatted_query: SqlQuery,
```

Comment:
This comment is helpful but it might be better for this to happen earlier in the file? It's easy to miss when starting with `execute_query`. Or maybe this function should come first, or the comment could be in both places? I'm not sure what makes sense here.

Reply:
ok

Reply:
it is in both places now.